app/services/immediate_email_generation_service.rb

```diff
@@ -54,21 +54,14 @@ def subscription_contents
       .where(email: nil)
   end
 
-  def build_many_emails(subscription_contents)
-    subscription_contents.map do |subscription_content|
-      build_email(subscription_content: subscription_content)
-    end
-  end
-
-  def build_and_insert_emails(subscription_contents)
-    emails = build_many_emails(subscription_contents)
-    Email.import!(emails)
-  end
-
-  def build_email(subscription_content:)
+  def import_emails(subscription_contents)
     ImmediateEmailBuilder.call(
-      subscriber: subscription_content.subscription.subscriber,
-      content_change: subscription_content.content_change
+      subscription_contents.map do |subscription_content|
+        {
+          subscriber: subscription_content.subscription.subscriber,
+          content_change: subscription_content.content_change,
+        }
+      end
```

Comment:
I'd suggest that it might be more readable to have the map result assigned to a variable and then passed in to the call, gets a little confusing within the method arguments.